test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/security/SecureRandomTest.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * scalajs-fake-insecure-java-securerandom (https://github.com/scala-js/scala-js-fake-insecure-java-securerandom)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.security
+
+import org.junit.Test
+import org.junit.Assert._
+
+import java.security.SecureRandom
+
+class SecureRandomTest {
+  @Test def nextBytes(): Unit = {
+    val random = new SecureRandom()
+
+    val buffer = new Array[Byte](24)
+    assertTrue("confidence check", buffer.count(_ == 0) == 24)
+
+    random.nextBytes(buffer)
+
+    // the likelihood of this failing should be one in 4 billion
+    assertTrue("nextBytes() must fill the provided buffer", buffer.count(_ == 0) < 4)
```

Comment:
Is it worth doing this check? Feels like the difference check should be enough.

Reply:
The difference check does not protect against accidentally filling only a prefix of the array, for example. This ensures that all bytes are covered.